torch/ao/quantization/qconfig.py

```diff
@@ -253,7 +253,7 @@ def get_default_qconfig(backend='x86', version=0):
             qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=False),
                               weight=default_weight_observer)
         elif backend == 'onednn':
-            if not torch.cpu._is_cpu_support_vnni():
+            if not torch.cpu._is_cpu_support_avx512_vnni():
```

Comment:
I guess the caller here actually means "vnni" regardless of whether it is avx2_vnni or avx512_vnni. I would suggest we keep this API unchanged while only fix the implementation to reflect its intention, i.e., either avx2_vnni or avx512_vnni supported means vnni is supported.

Reply:
Okay, let keep using the origian name.